DangoPlop/Assets/Scripts/PowerupParent.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PowerupParent : MonoBehaviour {
+
+	static private string playerTag = "Player";
+	public float timeLandedToDestroy = 3;
+	public float groundYPosition;
+
+	// Use this for initialization
+	void Start () {
+
+	}
+	
+	// Update is called once per frame
+	void Update () {
```

Comment:
@xiaogz Is this powerup falling at a constant speed? I think it's better if the powerup floats down at a constant speed, not following physics.